index.html

```diff
@@ -62,12 +59,11 @@
           </div>
           <div class="date-info">2022.04.14(MON) ~ (TUE)</div>
           <div class="location">
-            <div class="locate fs-5">@Nations of the World, Allow your citizens to learn Web Development!</div>
+            <div class="locate fs-5">@Starting this date, come and join to learn Web Development!</div>
           </div>
         </section>
       </section>
 
-      <!-- program -->
       <section id="program" class="program-bg">
         <div class="container-md p-0 my-0 mx-auto">
           <h3 class="cocogoose text-white text-center h4 fw-bold pt-5">Main Program</h3>
```

Comment:
- [x] When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hove mouse over them.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157689981-91c089cf-d233-49a4-b0d5-3e74f7a6d17b.png)